indexing-service/src/main/java/org/apache/druid/indexing/overlord/http/OverlordResource.java

```diff
@@ -422,6 +428,69 @@ public Response getWorkerConfig()
     return Response.ok(workerConfigRef.get()).build();
   }
 
+  /**
+   * Gets the total worker capacity of varies states of the cluster.
+   */
+  @GET
+  @Path("/totalWorkerCapacity")
+  @Produces(MediaType.APPLICATION_JSON)
+  @ResourceFilters(ConfigResourceFilter.class)
+  public Response getTotalWorkerCapacity()
+  {
+    // Calculate current cluster capacity
+    int currentCapacity;
+    Optional<TaskRunner> taskRunnerOptional = taskMaster.getTaskRunner();
+    if (!taskRunnerOptional.isPresent()) {
+      // Cannot serve call as not leader
+      return Response.status(Response.Status.SERVICE_UNAVAILABLE).build();
+    }
+    TaskRunner taskRunner = taskRunnerOptional.get();
+    Collection<ImmutableWorkerInfo> workers;
+    if (taskRunner instanceof WorkerTaskRunner) {
+      workers = ((WorkerTaskRunner) taskRunner).getWorkers();
+      currentCapacity = workers.stream().mapToInt(workerInfo -> workerInfo.getWorker().getCapacity()).sum();
+    } else {
+      log.debug(
+          "Cannot calculate capacity as task runner [%s] of type [%s] does not support listing workers",
+          taskRunner,
+          taskRunner.getClass().getName()
+      );
+      workers = ImmutableList.of();
+      currentCapacity = -1;
+    }
+
+    // Calculate maximum capacity with auto scale
+    int maximumCapacity;
+    if (workerConfigRef == null) {
+      workerConfigRef = configManager.watch(WorkerBehaviorConfig.CONFIG_KEY, WorkerBehaviorConfig.class);
+    }
+    WorkerBehaviorConfig workerBehaviorConfig = workerConfigRef.get();
+    if (workerBehaviorConfig == null) {
+      // Auto scale not setup
+      log.debug("Cannot calculate maximum worker capacity as worker behavior config is not configured");
```

Comment:
nit: similar comment to line 453 and other debug statements in this funciton.
```suggestion
      log.info("Cannot calculate maximum worker capacity as worker behavior config is not configured");
```